wfl/fit/ace.py

```diff
@@ -65,30 +68,98 @@ def fit(fitting_configs, ACE_name, params, ref_property_prefix='REF_',
         contructor to be used to run fitting in separate queued job
     ACE_FIT_JULIA_THREADS: used to set JULIA_NUM_THREADS for ace_fit.jl, which will use julia multithreading (LSQ assembly)
     ACE_FIT_BLAS_THREADS: used by ace_fit.jl for number of threads to set for BLAS multithreading in ace_fit
+
     """
+    # why can't we accept no prop prefix? 
```

Comment:
I suspect I just copied this from GAP fit or something.  The problem is that the ASE extxyz reader interprets plain `"energy"`, etc, as quantities to put in a `SinglePointCalculator`, rather than plain info fields. That's a hassle, because those are accessed in a qualitatively different way, and things like the database modification routines (that are used by gap-rss to make the per-config sigmas) need to handle it differently.  Also, it means that if you ever calculate something with these configurations (e.g. to compute a fitting error) the calculated values override the existing reference values.  Basically, it's to avoid messy ASE behavior.